scripts/Forestry.zs

```diff
@@ -77,6 +77,10 @@ for tomato in <ore:cropTomato>.items {
 #mods.forestry.Carpenter.addRecipe(<minecraft:redstone> * 9, [[<minecraft:redstone_block>]], 30);
 #mods.forestry.Carpenter.addRecipe(<minecraft:gold_ingot>, [[<minecraft:gold_block>]], 30, <liquid:for.honey> * 100);
 #mods.forestry.Carpenter.addRecipe(<minecraft:redstone_block>, [[<minecraft:redstone>,<minecraft:redstone>,<minecraft:redstone>],[<minecraft:redstone>,<minecraft:redstone>,<minecraft:redstone>],[<minecraft:redstone>,<minecraft:redstone>,<minecraft:redstone>]], 60, <liquid:water> * 200, <minecraft:stone>);
+mods.forestry.Carpenter.removeRecipe(<forestry:oak_stick>,<liquid:seed.oil>);
+mods.forestry.Carpenter.removeRecipe(<forestry:oak_stick>,<liquid:oliveoil>);
+mods.forestry.Carpenter.addRecipe(<forestry:oak_stick>*2, [[<ore:logWood>,null,null],[<ore:logWood>,null,null],[null,null,null]], 40, <liquid:seed.oil>*100);
+mods.forestry.Carpenter.addRecipe(<forestry:oak_stick>*2, [[<ore:logWood>,null,null],[<ore:logWood>,null,null],[null,null,null]], 40, <liquid:oliveoil>*100);
```

Comment:
Can the null parts be omitted to make the recipe shapeless?

Reply:
I checked the wiki and they can: https://docs.blamejared.com/1.12/en/Mods/Modtweaker/Forestry/Carpenter